### PR TITLE
added <a> tag button with mailto: and .button styling in CSS.

### PR DESCRIPTION
**Pull request by DaneCronin**
I wanted to place the <a> tag button code beneath all of the main content to position the button at the bottom but I was getting an error in VSCode about needing a parent element?  

The styling still needs to be more oval and rotated off-axis.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -157,6 +157,19 @@ a:focus {
...
 }
 
+.button {
+
+	padding: 25px 30px;
+	border-radius: 40px;
+	background-color: #ff7a7a;
+	color: #4e4e4e;
+	font-size: 14px;
+	font-family: soleil, sans-serif;
```
Comment (josephsap) [inline src/index.css:167]: The font doesn't match the gray text. I grabbed this from **.content p** in index.css and it visually matches. Could you replace  font-family: soleil, sans-serif; with what's below? Sorry, the CSS I copied is using [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).

```suggestion
	font-family: var(--font-alt);
```

**Comment by josephsap** [comment]
For the "tilt", look into CSS transform rotate. And the oval shape - this maybe? https://css-tricks.com/the-shapes-of-css/#aa-oval-shape

Don't worry about positioning it for now. It's strange, poorly laid-out code that I copied over because I wanted to do the least amount of work possible. How about I handle the positioning, and you make the font change and see if you can get it into an oval and rotated?

**Comment by DaneCronin** [comment]
Burning the midnight oil… I like it!

Yup, I noticed the font wasn’t a match and thought that other font with the variable in it might be the one, but I grabbed one that was legible to me. I will replace those and add the transform.  I had thought of that after and I think I thought of the way to make the button more of an oval shape with another border-radius property to target the corners differently than a uniform pixel radius.

Thanks man!

**Review by josephsap — APPROVED** [review]

**Comment by josephsap** [comment]
Thanks!

**Pull request merged**